frontend/src/metabase/nav/components/search/SearchBar/SearchBar.styled.tsx

```diff
@@ -25,24 +25,36 @@ export const SearchBarRoot = styled.div`
   }
 `;
 
-export const SearchInputContainer = styled.div<{ isActive: boolean }>`
+export const SearchInputContainer = styled.div<{
+  isActive: boolean;
+}>`
   display: flex;
   flex: 1 1 auto;
   align-items: center;
   position: relative;
 
-  background-color: ${props =>
-    props.isActive ? color("bg-medium") : color("bg-light")};
+  ${({ isActive }) => {
```

Comment:
I wanted to make sure that the background color doesn't change when the user hovers over the search bar when the search bar is already focused, it implies that there's more to click when there isn't